### PR TITLE
Merge in new cohort definitions from Denys

### PR DESCRIPTION
**Pull request by keesvanbochove**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ Long term outcomes of prostate cancer patients undergoing non-interventional man
 - Study end date: **-**
 - Protocol: please look in Teams
 - Publications: **-**
-- Diagnositcs explorer: **[Shiny App: Cohort diagnostics](https://data.ohdsi.org/PioneerWatchfulWaitingDiag/)**
+- Diagnositcs explorer: **[Shiny App: Cohort diagnostics] (https://data.ohdsi.org/PioneerWatchfulWaitingDiag/)**
```
Comment (MaximMoinat) [inline README.md:15]:
```suggestion
- Diagnositcs explorer: **[Shiny App: Cohort diagnostics](https://data.ohdsi.org/PioneerWatchfulWaitingDiag/)**
```

File: inst/sql/sql_server/104.sql [hunk 2]
```diff
@@ -400,14 +463,19 @@ FROM #qualified_events P
...
 (
-  select po.* 
+  select po.person_id, po.procedure_occurrence_id, po.procedure_date,DATEADD(d,1,po.procedure_date) as END_DATE, po.procedure_concept_id, po.visit_occurrence_id 
   FROM @cdm_database_schema.PROCEDURE_OCCURRENCE po
 JOIN #Codesets codesets on ((po.procedure_concept_id = codesets.concept_id and codesets.codeset_id = 10))
+--adding episode
+  union all
+  select ep.person_id, ep.episode_id, ep.episode_start_datetime, ep.episode_end_datetime,ep.episode_object_concept_id,null
+  from @cdm_database_schema.episode ep
+JOIN #Codesets codesets on ((ep.episode_object_concept_id = codesets.concept_id and codesets.codeset_id = 10))
```
Comment (MaximMoinat) [inline inst/sql/sql_server/104.sql:478]: Same episode issue here. If the episode table is needed in the definition, we should make a separate set of cohorts definitions for v6 databases.

**Review by MaximMoinat — COMMENTED** [review]
@denyskaduk 
We should re-evaluate how to accommodate the use of the episode table for the databases that have a v6 OMOP CDM. These changes will make the study fail for v5 OMOP CDM databases.

**Review by MaximMoinat — APPROVED** [review]
Looks good to me. @denyskaduk Please resolve the conflicts (some of the cohorts have changed in the meantime) and merge

File: R/CohortConstruction.R [hunk 3]
```diff
@@ -371,15 +371,17 @@ instantiateCohortSet <- function(connectionDetails = NULL,
...
       } else {
         sql <- SqlRender::render(sql,
                                  cdm_database_schema = cdmDatabaseSchema,
                                  vocabulary_database_schema = cdmDatabaseSchema,
                                  target_database_schema = cohortDatabaseSchema,
                                  target_cohort_table = cohortTable,
                                  target_cohort_id = cohorts$cohortId[i],
-                                 warnOnMissingParameters = FALSE)
+                                 warnOnMissingParameters = FALSE,
+                                 episodetable = FALSE)
```
Comment (MaximMoinat) [inline R/CohortConstruction.R:384]: @denyskaduk Do we expect a data source with episode tables to edit this source file? Shouldn't there be a parameter in the main run method?

Comment (denyskaduk) [reply R/CohortConstruction.R:384]: I'll work on the parametrization of the episode tables this week

**Comment by MaximMoinat** [comment]
We also need to pull in cohort changes from Atlas. Cohorts T2, T2a,  and all T3x have been changed.

File: inst/sql/sql_server/105.sql [hunk 4]
```diff
@@ -790,21 +717,26 @@ SELECT 0 as index_id, p.person_id, p.event_id
...
+select C.person_id, C.procedure_occurrence_id as event_id, C.procedure_date as start_date, DATEADD(d,1,C.procedure_date) as END_DATE,
+       C.procedure_concept_id as TARGET_CONCEPT_ID, C.visit_occurrence_id,
+       C.procedure_date as sort_date
 from 
 (
-  select de.* 
-  FROM @cdm_database_schema.DRUG_EXPOSURE de
-JOIN #Codesets codesets on ((de.drug_concept_id = codesets.concept_id and codesets.codeset_id = 14))
-) C
+  select po.*
```
Comment (MaximMoinat) [inline inst/sql/sql_server/105.sql:726]: Testing the code on NKR gave the error that both sides of the join should have an equal number of columns.

Should this be the following?
```suggestion
  select po.person_id, po.procedure_occurrence_id, po.procedure_date,DATEADD(d,1,po.procedure_date) as END_DATE, po.procedure_concept_id, po.visit_occurrence_id
```

Comment (bdemeulder) [reply inst/sql/sql_server/105.sql:726]: Hi all, can this issue be closed when Denys review the merge request # 45?

**Comment by bdemeulder** [comment]
@MaximMoinat @denyskaduk , let me know when I can merge this into episode branch and then merge the episode branch into the master.

**Comment by MaximMoinat** [comment]
I see my suggested fix has been applied. @denyskaduk Have been able to test the other queries as well? I want to make sure that this issue doesn't occur elsewhere.

**Comment by MaximMoinat** [comment]
Thanks @denyskaduk. Looks good, and I think we can merge @bdemeulder

**Pull request merged**